external/Makefile

```diff
@@ -77,6 +77,10 @@ include aws/Make.defs
 include aws/src/Make.defs
 endif
 
+ifeq ($(CONFIG_NETUTILS_WIFI),y)
```

Comment:
it checks the flag twice.

Reply:
It's better to remove CONFIG_NETUTILS_WIFI from /external/slsi_wifi/Make.defs